src/sentry/rules/conditions/level.py

```diff
@@ -58,7 +58,7 @@ def passes(self, event, state, **kwargs):
             return False
 
         desired_level = int(desired_level)
-        level = int(event.level)
+        level = int(event.group.level)
```

Comment:
Follow up to previous comment: I guess this point is also addressed in GH-3379 — the way that I personally would expect this condition to work is that it would be triggered if the *event* level matches the condition, not the level for the *group* that the level is associated with (which at this point may have changed levels anyway due to data races.)